### PR TITLE
Add Mixture Model parameters to docs

### PR DESCRIPTION
**Pull request by mamikonyana**
@raininthesun hope this helps a little

### CODE REVIEW

File: src/main/java/macrobase/analysis/stats/mixture/FiniteGMM.java [hunk 1]
```diff
@@ -38,6 +38,10 @@ public void train(List<Datum> data) {
         VariationalInference.trainMeanField(this, data, mixingComponents, clusters);
     }
 
+    /**
+     * @param datum
+     * @return log probability density of the given datum
+     */
     @Override
```
Comment (raininthesun) [inline src/main/java/macrobase/analysis/stats/mixture/FiniteGMM.java]: Is the probability density of the given datum corresponds to the likelihood of the datum? Say  the probability density of the given datum is 0.9, then does it mean that 90% this datum is not an outlier.

Comment (mamikonyana) [inline src/main/java/macrobase/analysis/stats/mixture/FiniteGMM.java]: this algorithm does not care how you think about outliers. probability density of the given datum being 0.9 means that there is `0.9 * ds` chance of a datum appearing within `ds` volume of the space surrounding the metrics of the datum.
Basically this number has the same meaning as the one you get from KDE

Comment (raininthesun) [inline src/main/java/macrobase/analysis/stats/mixture/FiniteGMM.java]: Got it! Thanks.

**Comment by raininthesun** [comment]
LGTM

**Pull request closed without merging**